crates/bevy_ecs/src/schedule/schedule.rs

```diff
@@ -1140,29 +1157,19 @@ impl ScheduleGraph {
 
     fn get_conflicting_systems(
         &self,
-        flat_results_disconnected: &Vec<(NodeId, NodeId)>,
-        ambiguous_with_flattened: &UnGraph,
+        flat_results_disconnected: &Vec<(SystemKey, SystemKey)>,
+        ambiguous_with_flattened: &UnGraph<NodeId>,
         ignored_ambiguities: &BTreeSet<ComponentId>,
     ) -> Vec<(SystemKey, SystemKey, Vec<ComponentId>)> {
         let mut conflicting_systems = Vec::new();
         for &(a, b) in flat_results_disconnected {
-            if ambiguous_with_flattened.contains_edge(a, b)
-                || self.ambiguous_with_all.contains(&a)
-                || self.ambiguous_with_all.contains(&b)
+            if ambiguous_with_flattened.contains_edge(a.into(), b.into())
```

Comment:
This `into()` is the same as the `NodeId::System` on the next line, right?  No preference for which way we do it, but it feels like they should do it the same way.  

```suggestion
            if ambiguous_with_flattened.contains_edge(NodeId::System(a), NodeId::System(b))
```